packages/dom-helper/lib/main.js

```diff
@@ -567,13 +569,26 @@ prototype.parseHTML = function(html, contextualElement) {
 var parsingNode;
 
 // Used to determine whether a URL needs to be sanitized.
-prototype.protocolForURL = function(url) {
-  if (!parsingNode) {
-    parsingNode = this.document.createElement('a');
-  }
+// In Node.js environments, we can't rely on the DOM correctly parsing URLs.
+// Instead, we'll fall back to Node's built-in URL parsing.
+if (typeof process === 'object' && process + '' === '[object process]') {
```

Comment:
`process` is likely not something you want to expose on the sandbox context just for an is node check.